### PR TITLE
rdkit_desc_update

### PR DESCRIPTION
**Pull request by stewu5**
1. include desc_list to allow selection of descriptor feature in rdkit
2. change test as length of rdkit DescriptorFeature becomes 208 (originally 200)

### CODE REVIEW

File: xenonpy/descriptor/fingerprint.py [hunk 1]
```diff
@@ -672,12 +703,21 @@ def __init__(self, n_jobs=-1,
...
             Default is None.
+        desc_list: string or list
+            List of descriptor names to be called in rdkit to calculate molecule descriptors.
+            If ``classic``, the full list of rdkit v.2020.03.xx is used. (length = 200)
+            Default is to use the latest list available in the rdkit. (length = 208 in rdkit v.2020.09.xx)
         """
         # self.arg = arg # arg[0] = radius, arg[1] = bit length
         super().__init__(n_jobs=n_jobs, on_errors=on_errors, return_type=return_type, target_col=target_col)
         self.input_type = input_type
-        nms = [x[0] for x in Descriptors._descList]
```
Comment (TsumiNa) [inline xenonpy/descriptor/fingerprint.py:679]: A little suggestion. Rename `Descriptors` to something like `ChemDescriptors` will be better for reading.
Modify `from rdkit.chem import Descriptors` to `from rdkit.chem import Descriptors as ChemDescriptors` will do this work.

**Review by TsumiNa — CHANGES_REQUESTED** [review]
See the inline comment.

Also, update the rdikt version in `.github/config/linux_win_env.yml` and `.github/config/macos.yml` to support new tests.

File: xenonpy/descriptor/fingerprint.py (hunk 1 above)
Comment (stewu5) [reply xenonpy/descriptor/fingerprint.py:679]: updated as you recommended to this branch

**Review by TsumiNa — APPROVED** [review]

**Comment by TsumiNa** [comment]
I have changed the rdkit version to 2020.09. Waiting for the test.

**Pull request merged**